Филачева Полина

polinafilacheva opened the pull request.

polinafilacheva committed:
начало верстки

polinafilacheva committed:
добавила стили

polinafilacheva committed:
начало вестки 320

polinafilacheva committed:
добавида media для 768px

evgeny-matveev commented on images/krug.svg:
```diff
@@ -0,0 +1,3 @@
+<svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Сделать через СВГ не ошибка, но еще этот кружок можно было бы нарисовать и в ЦСС.

polinafilacheva replied:
попыталась его вставить, не получилось, а потом просто про него забыла((

evgeny-matveev commented on index.html:
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
```
Сайт явно не английском :)

evgeny-matveev commented on index.html:
```diff
@@ -0,0 +1,33 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
А почему ты решила поставить этот мета-тег?

polinafilacheva replied:
Он сам у меня появляется когда я пользуюсь эметом, его лучше удалить?

evgeny-matveev commented on index.html:
```diff
@@ -0,0 +1,33 @@
+    </ul>
+  </header>
+  <section class="photo-news">
+   <img src="images/Book.jpg">
```
Названия файлов лучше писать прописными.

evgeny-matveev commented on index.html:
```diff
@@ -0,0 +1,33 @@
+    <h1>Самая темная чаща</h1>
+    <h2>Холли Блек</h2>
+    <div class="button">
+      <a class="one" href="+">Читать</a>
```
Зачем тут в href знак плюса?

evgeny-matveev commented on styles/style.css:
```diff
@@ -0,0 +1,192 @@
+   justify-content: space-around;
+   align-items: center;
+  }
+.menu {
```
Неровно :(

polinafilacheva committed:
внесла измененя

убрала лишний мета тег, поменяла язык, убрала + в href и убрала заглавную букву у картинки
